### ISSUE TITLE
Arrow point to a wrong position

### ISSUE THREAD

**Issue by ykying** [dhrebeniuk/GaugeMeterView#1]
After set GaugeMeterView's gaugeAngle, the arrow will point to a wrong position.
I try to fix it by change following code
```javascript
// FROM
let angle = CGFloat(value)*CGFloat.pi/180.0 + CGFloat.pi*CGFloat(0.5*gaugeAngle/(maxValue-gaugeAngle)) - CGFloat.pi
// TO
let angle = CGFloat(value)*CGFloat.pi/180.0 + CGFloat(gaugeAngle)*CGFloat.pi/180.0 - CGFloat.pi
```

**Comment by dhrebeniuk** [comment]
@ykying, hello may you send to me code when I can reproduce problem. Thanks

**Comment by ykying** [comment]
-> self.viewGauge.gaugeAngle = 0 //It point to a correct position
<img width="50%" alt="Screenshot 2019-09-05 at 09 41 29" src="https://user-images.githubusercontent.com/1370971/64305318-a081a700-cfc1-11e9-8dd8-8a753d8b066e.png">

-> self.viewGauge.gaugeAngle = 30 //Arrow point to a wrong position
<img width="50%" alt="Screenshot 2019-09-05 at 09 41 44" src="https://user-images.githubusercontent.com/1370971/64305350-c3ac5680-cfc1-11e9-9e0d-997318741b24.png">

One more finding, if I set the **gaugeValuesScale to 0.5**, the arrow position has same issue.
<img width="50%" alt="Screenshot 2019-09-05 at 09 46 07" src="https://user-images.githubusercontent.com/1370971/64305512-361d3680-cfc2-11e9-834a-db5f47e563f3.png">

**Comment by dhrebeniuk** [comment]
@ykying , Issue was fixed in 0.0.6 version. Thanks for reporting
